codespell_lib/data/dictionary_code.txt

```diff
@@ -62,7 +62,6 @@ rela->real
 rendir->render
 rendirs->renders
 ro->to, row, rob, rod, roe, rot,
-seeked->sought
```

Comment:
I think `seeked` was in the _code_ dictionary because it is an HTML event:
[HTML Audio/Video DOM seeked Event](https://www.w3schools.com/tags/av_event_seeked.asp)

Perhaps we should put `seeked` back here in the code dictionary.